services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java

```diff
@@ -456,8 +456,13 @@ private void handleRerouteRequest(CommandMessage message, Tuple tuple) throws Re
                 logger.warn("Reroute {} is unsuccessful: can't find new path.", flowId);
             }
 
-            PathInfoData pathInfoData = reroutedFlow != null
-                    ? FlowPathMapper.INSTANCE.map(reroutedFlow.getForward().getFlowPath()) : null;
+            PathInfoData pathInfoData =
+                    Optional.ofNullable(reroutedFlow != null ? reroutedFlow : flowService.getFlowPair(flowId))
```

Comment:
It differs from current implementation/API. Now we always return flow path (despite successful/failed reroute).